src/scripts/main.js

```diff
@@ -1 +1,22 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--menu-open');
+  } else {
+    document.body.classList.remove('page__body--menu-open');
+  }
+});
+
+const form = document.querySelector('.form');
+const inputs = document.querySelectorAll('.js-test');
+
+function handler(event) {
+  event.preventDefault();
+
+  inputs.forEach(input => {
+    input.value = '';
+  });
+}
+
+form.addEventListener('submit', handler);
```

Comment:
this part of your code don't work. U need to add .js-test class to all of your inputs

https://user-images.githubusercontent.com/103505670/188403212-38e85a5a-272d-4992-9b1a-8a320d48a7fd.mp4